debian/control

```diff
@@ -6,23 +5,32 @@ Maintainer: Go Compiler Team <pkg-golang-devel@lists.alioth.debian.org>
 Uploaders: Michael Stapelberg <stapelberg@debian.org>,
            Paul Tagliamonte <paultag@debian.org>,
            Tianon Gravi <admwiggin@gmail.com>
-Vcs-Browser: http://anonscm.debian.org/gitweb/?p=pkg-golang/golang.git
+Vcs-Browser: https://anonscm.debian.org/cgit/pkg-golang/golang.git
 Vcs-Git: git://anonscm.debian.org/pkg-golang/golang.git
-Build-Depends: debhelper (>= 7.4.10), bison, ed, mawk | awk, perl, netbase
+Build-Depends: debhelper (>= 7.4.10), bison, ed, mawk | awk, perl, netbase,
+               golang (>= 1:1.4~) | gccgo-5
 Build-Depends-Indep: po-debconf
 Standards-Version: 3.9.6
-Homepage: http://golang.org/
+Homepage: https://golang.org
 
 Package: golang-go
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends},
-	 golang-src (>= ${source:Version}),
-	 golang-go-linux-${go:Hostarch} (= ${source:Version})
-Replaces: golang-weekly-go, golang-weekly-tools, golang-tools
-Conflicts: golang-weekly-go, golang-weekly-tools, golang-tools
-Provides: go-compiler, golang-weekly-go, golang-weekly-tools, golang-tools
+Architecture: amd64 arm64 armel armhf i386 ppc64 ppc64el
+Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}
+Replaces: golang-src,
```

Comment:
You need to Breaks/Replaces: golang-golang-x-tools (<= 1:0.0~git20150716.0.87156cb+dfsg1-3) or something like that